fs/unstable_read_text_file.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright 2018-2025 the Deno authors. MIT license.
+
+import { mapError } from "./_map_error.ts";
+import type { ReadFileOptions } from "./unstable_types.ts";
+import { isDeno } from "./_utils.ts";
+import { readFile, readFileSync } from "./unstable_read_file.ts";
+
+/**
+ * @example Usage
```

Comment:
Can you also copy the main description of API from Deno API docs? https://docs.deno.com/api/deno/~/Deno.readTextFile

Reply:
Updated.